src/main/java/uk/gov/ons/ctp/response/sample/message/CollectionExerciseEndActivation.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.ons.ctp.response.sample.message;
+
+import static net.logstash.logback.argument.StructuredArguments.kv;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import libs.common.error.CTPException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.gcp.pubsub.support.BasicAcknowledgeablePubsubMessage;
+import org.springframework.cloud.gcp.pubsub.support.GcpPubSubHeaders;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.handler.annotation.Header;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.ctp.response.sample.representation.CollectionExerciseEndEventDTO;
+import uk.gov.ons.ctp.response.sample.service.CollectionExerciseEndService;
+
+@Component
+public class CollectionExerciseEndActivation {
+
+  private static final Logger LOG = LoggerFactory.getLogger(CollectionExerciseEndActivation.class);
+
+  @Autowired private ObjectMapper objectMapper;
+
+  @Autowired private CollectionExerciseEndService collectionExerciseEndService;
+
+  /** To process Collection exercise end from PubSub. */
+  @ServiceActivator(inputChannel = "collectionExerciseEndActivationChannel")
+  public void messageReceiver(
+      Message message,
+      @Header(GcpPubSubHeaders.ORIGINAL_MESSAGE) BasicAcknowledgeablePubsubMessage pubSubMsg) {
+    LOG.info(
+        "Receiving Collection exercise end event",
+        kv("PubSub ID", pubSubMsg.getPubsubMessage().getMessageId()));
```

Comment:
Could we log it as "messageId" rather than "PubSub ID", then it's consistent with other services